app/src/main/java/app/nexd/android/ui/helper/transcript/TranscriptViewModel.kt

```diff
@@ -227,7 +212,7 @@ class TranscriptViewModel(private val api: Api) : ViewModel() {
         val disposable = api.phoneControllerConverted(call.value?.sid, data)
             .observeOn(mainThread())
             .subscribeBy(
-                onNext = { resetData() },
+                onNext = { call.value = null },
```

Comment:
Because in the summary fragment. I still need first name to show some information ("thank you for translating ... call")
The rest of the data will be reset if the user wants to translate another call

Reply:
but we can simply reset everything direclty in transcriptCall() like it was before. We don't need to reset the call separately before.

Reply:
I see that fragment navigation is based on the null value. that is the issue here. we have to implement a proper progress here. i'm on it